styles.css

```diff
@@ -1,826 +1,980 @@
-  color: #fff;
-  text-align: center;
-  height: 56px;
-  font-size: 40px;
-  line-height: 56px;
-  margin: 20px 0;
-}
-
-.contact-content p {
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  color: #ebebff;
-  height: 136px;
-  font-size: 20px;
-  line-height: 28px;
-  font-weight: 4000;
-  text-align: center;
-}
-
-.name-container input {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 48px;
-  padding-left: 5px;
-}
-
-.mail-container input {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 48px;
-  padding-left: 5px;
-}
-
-.textarea-container textarea {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 170px;
-  padding-left: 5px;
-}
-
-button {
-  width: 140px;
-  height: 48px;
-  margin-left: 40px;
-  border: 1px solid #6070ff;
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  background: #fff;
-  border-radius: 8px;
-  color: #6070ff;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
+/* HTML5 Reset Stylesheets. */
+
+/* -----Header Section ---- */
+@media only screen and (min-width: 376px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 1rem;
+  }
+
+  .logo {
+    margin: 4px 199px 4px 12px;
+  }
+
+  .menu {
+    margin: 4px 24px;
+  }
+
+  nav {
+    display: none;
+  }
+
+  /* -----Header Section ---- */
+
+  .wrapper {
+    position: relative;
+    background: #e5e5e5;
+    height: auto;
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  section.intro {
+    width: 100%;
+    height: auto;
+    background-image: url("/img/header-shapes-mobile.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-color: #fff;
+    padding-bottom: 10px;
+    padding-top: 5px;
+  }
+
+  .title.description {
+    margin: 144px 24px 12px;
+    width: auto;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  h1.title {
+    margin-bottom: 12px;
+  }
+
+  p.description {
+    margin-bottom: 12px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .social.container {
+    margin: 12px 24px;
+    height: auto;
+  }
+
+  ul.social-icons {
+    display: flex;
+    margin-top: 12px;
+    align-items: center;
+  }
+
+  ul.social-icons li img {
+    margin-right: 19.01px;
+  }
+
+  /* "Works" section */
+
+  .works {
+    display: grid;
+    margin: 114px 24px;
```

Comment:
- Please, kindly reduce your margin as it makes the spaces between each card too big. please fix it to match the design as much as you can:
- 
![reduce](https://user-images.githubusercontent.com/59397159/187049729-09c7e796-784e-49ff-865e-0a921e92568f.png)